src/connector/src/parser/csv_parser.rs

```diff
@@ -44,8 +56,15 @@ pub struct CsvParser {
 }
 
 impl CsvParser {
-    pub fn new(delimiter: u8, has_header: bool) -> Result<Self> {
+    pub fn new(rw_columns: Vec<SourceColumnDesc>, parser_config: CsvParserConfig) -> Result<Self> {
+        let CsvParserConfig {
+            delimiter,
+            has_header,
+        } = parser_config;
+
+        // tracing::info!("rw_columns: {:?}", rw_columns);
```

Comment:
If it's too verbose, give it another level or just remove the comments.

Reply:
removed